app/models/vha_facility_adapter.rb

```diff
@@ -121,7 +121,7 @@ def self.mh_clinic_phone(attrs)
   end
 
   def self.services_date(attrs)
-    Time.at(attrs['OutpatientServicesDataDate'] / 1000).utc.to_date.iso8601 if attrs['OutpatientServicesDataDate']
+    Date.strptime(attrs['FacilityDataDate'], '%m-%d-%Y').iso8601 if attrs['FacilityDataDate']
```

Comment:
is it ok if the result of this could be nil or would it be better to have a default fallback for when 'FacilityDataDate' is not present. Also, will FacilityDataDate always be coercible? Maybe there should be a rescue statement?